doc/conf.py

```diff
@@ -239,7 +240,7 @@
 event_sig_re = re.compile(r'([a-zA-Z-]+)\s*\((.*)\)')
 
 
-def parse_event(env, sig, signode):
+def parse_event(env, sig: str, signode) -> str:
```

Comment:
I'd say: either a full typing or none, because I'd rather have some `env` and `signode` completion in general.